src/web/services/log.service.ts

```diff
@@ -17,6 +17,7 @@ import { HttpRequestService } from './http-request.service';
   providedIn: 'root',
 })
 export class LogService {
+  static readonly LOG_DATE_TIME_FORMAT: string = 'ddd, DD MMM YYYY hh:mm:ss A';
```

Comment:
Noted, I will move the constant into `InstructorAuditLogsPageComponent`instead, since logically that seems to be the most relevant class. Do let me know if there is a better place to put the constant!

Reply:
`api-const.ts` is actually a worse choice because that file is to transmit some data from back-end, and back-end has no business with how things are displayed.

Defining in the page component itself works.

Reply:
alright, I'll add it there then 👍

Reply:
Oops, adding it there results in circular dependency. Is that ok or should I just keep the date time format as separate strings?